src/pages/login/components/LoginForm.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { FlexBox } from '~/components/Box';
+import { useSelector } from '~/store/hooks';
+import { FormRow } from '~/components/forms/FormRow';
+import { Button } from '~/components/Button';
+import { FormProvider, useForm } from 'react-hook-form';
+
+export interface formData {
+  username: string;
+  password: string;
+}
+
+interface Props {
+  onSubmit: (data: formData) => void;
+}
+
+export const LoginForm: React.FC<Props> = ({ onSubmit }) => {
+  const { isError, isLoading } = useSelector(state => state.user);
+  const methods = useForm();
+
+  return (
+    <FormProvider {...methods}>
+      <form onSubmit={methods.handleSubmit(onSubmit)}>
+        <FormRow label="Nazwa Uzytkownika" formId={'username'} type={'text'} placeholder={'Nazwa Uzytkownika'} />
+        <FormRow label="Haslo" formId={'password'} type={'password'} placeholder={'Haslo'} />
+        <FlexBox justifyContent="flex-end">
+          <Button type={'submit'} text={'Zaloguj'} width="20%" />
+        </FlexBox>
+        {isError && <h5>ERROR!!!</h5>}
+        {isLoading && <h5>Loading....</h5>}
```

Comment:
Same with loading indicator - better solution would be adding a loader somewhere in the form, most commonly now such loaders is placed inside submit button